apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -323,9 +340,24 @@ export default function Success(props: PageProps) {
       }
       return t(`needs_to_be_confirmed_or_rejected${titleSuffix}`);
     }
+    //Scenerio when admin/owner views the bookings details for the team members.
+    if (teamId && memberId) {
+      if (bookingInfo.user && membershipRole) {
+        const roleBasedAttendees =
+          membershipRole === "OWNER" || membershipRole === "ADMIN"
+            ? bookingInfo.attendees.map((attendee) => attendee.name || attendee.email).join(", ")
+            : "You";
```

Comment:
Wouldn't this also change it in case user is part of the booking but is also team admin/owner. 
What I would suggest is just check if current session user is present in booking host or part of attendees. If yes then you can show "You are meeting with .....", if not then we want to name all attendees and hosts.

Reply:
okay this approach seems more appropriate. Will update it

Reply:
I don’t think listing all the attendees' names or emails in a comma-separated format at the top will look good, especially since a meeting typically has many attendees

Reply:
@anikdhabal Instead of attendees names or emails, can you please suggest something ?